### DISCUSSION TITLE
Pull back related data in GraphQL over multiple fields

### DISCUSSION THREAD

**Discussion by jimogden** [gatsbyjs/gatsby#25791] [Help]
I have a similar requirement to the one outlined by jf-home in question https://github.com/gatsbyjs/gatsby/issues/25512.  Following the advice provided by @pieh, I can pull back related data for one field but how would I go about implementing the same for additional fields? 

 System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.449.0
  npmPackages:
    gatsby: ^2.0.76 => 2.21.22
    gatsby-background-image: ^0.9.19 => 0.9.19
    gatsby-image: ^2.0.20 => 2.4.3
    gatsby-plugin-catch-links: ^2.3.9 => 2.3.9
    gatsby-plugin-html-attributes: ^1.0.5 => 1.0.5
    gatsby-plugin-manifest: ^2.4.16 => 2.4.16   
    gatsby-plugin-offline: ^2.2.10 => 2.2.10    
    gatsby-plugin-react-helmet: ^3.3.8 => 3.3.8
    gatsby-plugin-sharp: ^2.6.16 => 2.6.16
    gatsby-plugin-svgr: ^2.0.2 => 2.0.2
    gatsby-remark-images: ^3.3.16 => 3.3.16
    gatsby-remark-relative-images: ^0.3.0 => 0.3.0
    gatsby-source-filesystem: ^2.3.16 => 2.3.16
    gatsby-transformer-remark: ^2.8.22 => 2.8.22
    gatsby-transformer-remark-frontmatter: ^0.1.0 => 0.1.0
    gatsby-transformer-sharp: ^2.5.9 => 2.5.9
  npmGlobalPackages:
    gatsby-cli: 2.12.45

**Comment by jimogden** [comment]
Just wondered if there is any additional info you might need?

Thanks!

**Comment by vladar** [comment]
Sorry, this issue slipped through under the radar somehow. Can you clarify what doesn't work for you with some code example / reproduction? And what you've tried already?
